crates/adapters/src/controller/error.rs

```diff
@@ -129,7 +152,7 @@ pub enum ControllerError {
     /// new valid inputs after an error.
     ParseError {
         endpoint_name: String,
-        error: AnyError,
+        error: String,
```

Comment:
Why is this instance of error a String whereas the followings ones are AnyError?

Reply:
Good point.  Problem is,  `AnyError` is not serializable.  The downside of `String` on the other hand is that it loses internal error structure.  I plan to make them all `String` for now.  With some extra work we can do better in the future if needed.